eipw-lint/src/lib.rs

```diff
@@ -574,17 +577,19 @@ where
     pub fn to_iters(
         &self,
     ) -> Options<
-        impl Iterator<Item = Box<dyn Modifier>> + '_,
-        impl Iterator<Item = (&'_ str, Box<dyn Lint>)>,
+        Box<ModifierIterator<'_>>, // Use boxed trait objects
+        Box<LintIterator<'_>>,     // Use boxed trait objects
```

Comment:
What's the advantage to using a boxed iterator over an impl Trait?

Reply:
Makes the compiler happier :)